cmd/manta-init.js

```diff
@@ -547,6 +554,22 @@ var pipelineFuncs = [
                 )
             );
         }
+        var providedOptions = Object.keys(ARGV);
```

Comment:
Perhaps a comment for this section clarifying that we are validating that all given options are actually expected ones.

Reply:
Can do.